tests/testthat/test-pqn.R

```diff
@@ -0,0 +1,96 @@
+test_that("Proximal Quasi-Newton: gaussian, n>p case", {
+  library(SLOPE)
+  set.seed(1)
+
+  n = 100
+  p = 10
+
+  d <- solvers::randomProblem(n, p, response="gaussian", density = 0.5)
+  
+  admm_solvers <- ADMM(d$x, d$y, family="gaussian", alpha=c(1.0,0.005), opt_algo="nr")
+  pqn_solvers <- PN(d$x, d$y, family="gaussian", alpha=c(1.0,0.005), hessian_calc="lbfgs")
+  expect_equivalent(coef(admm_solvers), coef(pqn_solvers), tol = 1e-2)
+
+})
```

Comment:
Yes. I'll update this.

Reply:
There are a few problems arising from combining them into same test. Due to varying precision, it is difficult to adjust hyperparameters so that tests pass. For instance, I have to force fista to run for more iterations so that it matches admm's output. However, that still doesn't match output of pn. 
I think it is better that tests stay as the way they are right now.

Reply:
All right, no problem.